### ISSUE TITLE
Info-Icon zur Beschreibung von Veranstaltungsfilteroption "aktuell" analog zum Info-Icon bei Kontakt

### ISSUE THREAD

**Issue by Bensk1** [Strassengezwitscher/Strassengezwitscher#241]
Text: "Es werden kommende und vergangene Veranstaltungen der letzten 30 Tage angezeigt."

**Comment by mathebox** [comment]
Also als Tooltip?

**Comment by humuzz** [comment]
Kann man diesen sowie den zweiten Hinweis so einstellen, dass das Hinweisfeld am Bildschirmrand endet, also nicht vom Rand abgeschnitten wird?

**Comment by Bensk1** [comment]
Ja, leider liegt momentan ein Fehler in einem Software-Paket vor, auf das wir aufbauen. Dieser Fehler führt zu dem Abschneiden. Wir kümmern uns aber darum.
